CTP/shell/src/com/navercorp/cubridqa/shell/deploy/DeployOneNode.java

```diff
@@ -199,44 +197,17 @@ private void deploy_linux_by_cubrid_common() {
 		}
 	}
 		
-	private void deploy_cubrid_common() {
-	
-		ShellInput scripts = new ShellInput();
-		scripts.addCommand("echo 'BEGIN TO UPGRADE cubrid_common'");
-		scripts.addCommand("cd $HOME/cubrid_common");
-		scripts.addCommand("sh upgrade.sh 2>&1");
-		
-		String result;
-		try {
-			result = ssh.execute(scripts);
-			log.println(result);
-		} catch (Exception e) {
-			log.print("[ERROR] " + e.getMessage());
-		}
-	}
-	
-	private void deploy_core_analyzer() {
-		
-		ShellInput scripts = new ShellInput();
-		scripts.addCommand("echo 'BEGIN TO UPGRADE core_analyzer'");
-		scripts.addCommand("cd $HOME/core_analyzer");
-		scripts.addCommand("sh upgrade.sh 2>&1");
-		
-		String result;
-		try {
-			result = ssh.execute(scripts);
-			log.println(result);
-		} catch (Exception e) {
-			log.print("[ERROR] " + e.getMessage());
-		}
-	}
-	
-	private void deploy_cqt() {
+	private void deploy_ctp() {
+		String enableSkipUpgrade = context.getEnableSkipUpgrade();
+		String branchName = context.getCtpBranchName();
+		String ctpHome = context.getToolHome();
 		
 		ShellInput scripts = new ShellInput();
-		scripts.addCommand("echo 'BEGIN TO UPGRADE CQT'");
-		scripts.addCommand("cd $HOME/CQT");
-		scripts.addCommand("sh upgrade.sh 2>&1");
+		scripts.addCommand("echo 'BEGIN TO UPGRADE CTP'");
+		scripts.addCommand("export SKIP_UPGRADE=" + enableSkipUpgrade);
+		scripts.addCommand("export CTP_BRANCH_NAME=" + branchName);
+		scripts.addCommand("cd " + ctpHome);
```

Comment:
The ctpHome in test node is different from controller node.

So in test node, just use $HOME/CTP.